types/redux-injectable-store/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for redux-injectable-store 1.0
+// Project: https://github.com/lelandrichardson/redux-injectable-store#readme
+// Definitions by: Daniel Perez Alvarez <https://github.com/unindented>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Reducer, Store, StoreEnhancer } from "redux";
+
+export interface InjectableStore<S> extends Store<S> {
+    inject(namespace: string, reducer: Reducer<any>): void;
```

Comment:
should these be generic instead of defaulting to `any`? i.e. `inject<T>(namespace: string, reducer: Reducer<T>: void;`

Reply:
Fair enough. I'll make that change.

Reply:
Done.

Reply:
Guess who misunderstands generics... sorry, about that.

https://github.com/Microsoft/dtslint/blob/master/docs/no-unnecessary-generics.md

Reply:
Haha, I hadn't read that advice either. I'll go back to the original.

Reply:
Done.